ssm/message_directory.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """This module contains the MessageDirectory class."""
+from __future__ import print_function
```

Comment:
Is this import needed? It doesn't seem to be used in this file/class?

Reply:
I wanted to avoid regression if a print statement gets added.

Reply:
In that case wouldn't the Python 3.7 Travis build fail?

And if a Python 3 style print function gets added to the code later, wouldn't the Python 2.7 build fail if this line was omitted where it was then needed.

Reply:
> In that case wouldn't the Python 3.7 Travis build fail?

Only if there's a test that executes the code.

> And if a Python 3 style print function gets added to the code later, wouldn't the Python 2.7 build fail if this line was omitted where it was then needed.

It wouldn't as Python 2.7 is able to ignore the parentheses and it would treat it as just a print statement, but I'd rather it was treated it the same as Python 3, hence the import.

The general idea is to make whatever code is in the file behave the same under Python 2.7 or 3.

Reply:
> Python 2.7 is able to ignore the parentheses and it would treat it as just a print statement

Ah yes, it will. Fair enough.